db/migration/1688372371221-AddPresentationToVariables.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddPresentationToVariables1688372371221
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE variables
+                ADD COLUMN schemaVersion INT NOT NULL DEFAULT 1,
+                ADD COLUMN processingLevel ENUM('minor', 'major'),
+                ADD COLUMN processingLog JSON,
+                ADD COLUMN titlePublic TEXT,
+                ADD COLUMN titleVariant TEXT,
+                ADD COLUMN producerShort TEXT,
+                ADD COLUMN citationInline TEXT,
+                ADD COLUMN descriptionShort TEXT,
+                ADD COLUMN descriptionFromProducer TEXT,
+                -- ADD COLUMN topicTagsLinks JSON,
+                ADD COLUMN keyInfoText JSON,
+                ADD COLUMN processingInfo TEXT,
+                ADD COLUMN licenses JSON,
+                ADD COLUMN presentationLicense JSON,
+                ADD COLUMN grapherConfigETL JSON;`
```

Comment:
Are we set on having two `grapherConfig` objects on the variable? If so, could we rename the other one, to say what it's meant for?

If the ETL one is always the base, perhaps it could be `grapherConfigBase` and `grapherConfigOverlay`, or `grapherConfigETL` and `grapherConfigAdmin`?

Reply:
Yes I think it's cleaner if we have two grapher objects for now. I think grapherConfigAdmin is a name that is easier to understand. 

My current best guess is that we'll want to use grapherConfigETL as our primary config. grapherConfigAdmin could be used as the place where we can store the copy of currently existing single variable grapher charts. This way, the grapherConfigAdmin can be filled once for a lot of variables, be written to from the Variable Annotation tool in the admin and is going to be immune from overwrites in the ETL. In the medium term we can then hopefully curate all the grapher config changes in the ETL and then evenutally drop grapherConfigAdmin.